src/main/java/no/hyper/imagecrop/ImageTray.java

```diff
@@ -0,0 +1,205 @@
+package no.hyper.imagecrop;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Point;
+import android.graphics.Rect;
+import android.support.v4.view.GestureDetectorCompat;
+import android.util.AttributeSet;
+import android.util.Log;
+import android.view.GestureDetector;
+import android.view.MotionEvent;
+import android.view.ScaleGestureDetector;
+import android.view.View;
+
+/**
+ * Created by jean on 07/10/15.
+ */
+class ImageTray extends View {
+
+    private Context context;
+    private Bitmap picture;
+    private Point middle;
+    private Paint bitmapPaint;
+    private Rect cropRect;
+    private int screenWidth;
+    private int screenHeight;
+    private int cropSize = 500;
```

Comment:
Could we move this value into a static constant on ImageCropper or something, so that we don't need to hardcode the number_500_ everywhere for the defaults? It'll make the code a teensy bit easier to understand, and there will be less places to refactor if we want to change the value later. :smile: